Makefile

```diff
@@ -1,9 +1,14 @@
 export SERVICE_NAME=xds-relay
 
+SOURCE_FILES?=$$(go list ./... | grep -v integration-tests)
```

Comment:
may be adding a comment about why integration tests need to be added specially will help.

Reply:
will do.